feature/extrasA/wildfly-feature-pack-build.xml

```diff
@@ -0,0 +1,43 @@
+<!--
+  ~ JBoss, Home of Professional Open Source.
```

Comment:
Maybe use the WFC project license? Likewise for the other `feature-pack-build.xml `files.

Reply:
+1 and I wish we could introduce mycila license plugin which these kind of issues can be checked and fixed automatically.

Reply:
We do have the [license-maven-plugin](https://github.com/jamesnetherton/wildfly-liquibase/blob/master/pom.xml#L187).

Reply:
Apparently, it does not protest on missing/wrong license headers. I know and prefer com.mycila:license-maven-plugin.

Reply:
I think it does. Via the `check-file-header` goal, I use it in one of my personal projects.

Reply:
`check-file-header` does not seem to be active by default.